.github/workflows/publish-hotfix-release.yml

```diff
@@ -0,0 +1,267 @@
+    needs: [ prepare-hotfix-release, release-version ]
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      packages: write
+      pages: write
+
+    if: needs.release-version.outputs.RELEASE_VERSION
+    steps:
+      - name: Export RELEASE_VERSION env
+        run: |
+          echo "RELEASE_VERSION=${{ needs.release-version.outputs.RELEASE_VERSION }}" >> $GITHUB_ENV
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Install Helm
+        uses: azure/setup-helm@v4
+        with:
+          version: v3.8.1
+      - name: Package helm, update index.yaml and push to gh-pages
+        run: |
+          # Prepare git env
+          git config user.name "eclipse-tractusx-bot"
+          git config user.email "tractusx-bot@eclipse.org"
+
+          # Package all charts
+          find charts -name Chart.yaml -not -path "./edc-tests/*" | xargs -n1 dirname | xargs -n1 helm package -u -d helm-charts
+
+          git checkout gh-pages || git checkout -b gh-pages
+          git pull --rebase origin gh-pages
+
+          # Generate helm repo index.yaml
+          helm repo index . --merge index.yaml --url https://${GITHUB_REPOSITORY_OWNER}.github.io/${GITHUB_REPOSITORY#*/}/
+
+          # Commit and push to gh-pages
+          git add index.yaml helm-charts
+          git commit -s -m "Release ${{ env.RELEASE_VERSION }}"
+
+          git push origin gh-pages
+
+  # Release: GitHub tag & release; Merges back releases into main; Starts a new development cycle;
+  github-release:
+    name: Publish new github release
+    needs: [ prepare-hotfix-release, release-version, maven-release, docker-release, helm-release ]
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      packages: write
+      pages: write
+      pull-requests: write
+    if: needs.release-version.outputs.RELEASE_VERSION
+    steps:
+      - name: Export RELEASE_VERSION env
+        run: |
+          echo "RELEASE_VERSION=${{ needs.release-version.outputs.RELEASE_VERSION }}" >> $GITHUB_ENV
+      - uses: actions/checkout@v4
+        with:
+          # 0 to fetch the full history due to upcoming merge of releases into main branch
+          fetch-depth: 0
+      - name: Create Release Tag
+        id: create_release_tag
+        run: |
+          # Prepare git env
+          git config user.name "eclipse-tractusx-bot"
+          git config user.email "tractusx-bot@eclipse.org"
+
+          # informative
+          git branch -a
+          git tag
+
+          # Create & push tag
+          git tag --force ${{ env.RELEASE_VERSION }}
+          git push --force origin ${{ env.RELEASE_VERSION }}
+      - name: Create GitHub Release
+        uses: ncipollo/release-action@v1
+        with:
+          generateReleaseNotes: true
+          tag: ${{ env.RELEASE_VERSION }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+          makeLatest: false
+          removeArtifacts: true
+
+  publish-to-swaggerhub:
```

Comment:
the job to publish the API spec to GH Pages is missing

Reply:
This was a copy of the existing publish-new-release workflow. That workflow did not contain the publication to GH pages so I left it out.
Publishing to GH pages started with 0.8.0. Does it make sense to have this hotfix there?